Mathlib/GroupTheory/MonoidLocalization/Basic.lean

```diff
@@ -949,11 +945,7 @@ of the induced maps equals the map of localizations induced by `l ∘ g`."]
 theorem map_map {A : Type*} [CommMonoid A] {U : Submonoid A} {R} [CommMonoid R]
     (j : LocalizationMap U R) {l : P →* A} (hl : ∀ w : T, l w ∈ U) (x) :
     k.map hl j (f.map hy k x) = f.map (fun x ↦ show l.comp g x ∈ U from hl ⟨g x, hy x⟩) j x := by
--- Porting note: Lean has a hard time figuring out what the implicit arguments should be
--- when calling `map_comp_map`. Hence the original line below has to be replaced by a much more
--- explicit one
---  rw [← f.map_comp_map hy j hl]
-  rw [← @map_comp_map M _ S N _ P _ f g T hy Q _ k A _ U R _ j l hl]
+  rw [← f.map_comp_map (k := k) hy j hl]
```

Comment:
The concern about having to specify `k` still applies here, right?

Reply:
I wrote a new porting note about specifying `k` explicitly. Although I think it is a minor concern.